plugins/module_utils/copy.py

```diff
@@ -216,6 +228,87 @@ def copy_vsam_ps(src, dest):
     return rc, out, err
 
 
+def copy_asa_uss2mvs(src, dest):
+    """Copy a file from USS to an ASA sequential data set or PDS/E member.
+
+    Arguments:
+        src: {str} -- Path of the USS file
+        dest: {str} -- The MVS destination data set or member
+
+    Returns:
+        boolean -- The return code after the copy command executed successfully
+        str -- The stdout after the copy command executed successfully
+        str -- The stderr after the copy command executed successfully
+    """
+    module = AnsibleModuleHelper(argument_spec={})
+
+    oget_cmd = 'tsocmd "OGET \'{0}\' \'{1}\'"'.format(src, dest)
```

Comment:
Some of the other new copy routines perform a validation on src and dest, this method `copy_asa_uss2mvs` does not perform the below, is it needed for our security scans? 

```
    src = _validate_data_set_name(src)
    dest = _validate_path(dest)
```

Reply:
No is not, executed Appscan and Mend security scans and there are no issues, we currently have no validations for mvs data set paths.